io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.atari800.Atari800
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: atari800
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all	
+  - --socket=pulseaudio
+  - --filesystem=home
+
+cleanup:
+  - '/bin/sdl-config'
+  - '/include'
+  - '/lib/pkgconfig'
+  - '*.a'
+  - '*.la'
+
+modules:
+  - name: libsdl
+    sources:
+      - type: archive
+        url: https://libsdl.org/release/SDL-1.2.15.tar.gz
+        sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
+      - type: patch
+        path: patch/sdlpatch.txt
+        strip-components: 1
+  - name: atari800
+    config-opts:
+      - --with-video=sdl
+      - --with-sound=sdl
+      - --enable-pngcodec
+      - --with-mp3=lame
+    post-install:
+      - install -Dm 0644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      # Access SVG icon inside local copy of git repo
+      - install -Dm 0644 ${FLATPAK_BUILDER_BUILDDIR}/data/atari2.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+    sources:
+      - type: git
+        url: https://github.com/atari800/atari800.git
+        tag: ATARI800_5_0_0
+        commit: c0f94a7bf15543d8feb87ba046c456087c9d8227
+      - type: dir
+        path: misc
```

Comment:
Ideally icon, appdata and desktop files should come from https://github.com/atari800/atari800.git If e.g. desktop file is already there but needs changes, `desktop-file-edit` can be used to modify specific fields during build.

Reply:
there is a svg and png there as I mentionned in the review.

Reply:
also it's better to reference files individually than use `dir`.

Reply:
Removed 'dir'.